src/selenium/generatorPage.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable no-empty-label */
+'use strict';
+
+const By = require('selenium-webdriver').By;
+const until = require('selenium-webdriver').until;
+
+const GeneratorPage = {
+  init: function(webdriver) {
+    this.driver = webdriver;
+  },
+
+  visit: function(url) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces but found 4. (indent)](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)